docs/guides/automating-a-process-using-bpmn.md

```diff
@@ -0,0 +1,103 @@
+
+If you're new to BPMN elements and their capabilities, also visit our [BPMN symbol documentation](https://camunda.com/bpmn/reference/).
+
+### BPMN in action
+
+Using these elements, we can build out our process in baking a cake.
+
+Take the following steps:
+
+1. On our diagram, we've already been given an element as a start event in the shape of a circle. We can click on the circle, and then the wrench icon to adjust this element. For now, we'll keep it as a start event. Double click on the circle to add text.
+2. Drag and drop an arrow to the first task (the rectangle shape), or click the start event, and then click the task element to automatically attach it.
+3. Click on the task, then click on the wrench icon to declare it a user task, which will be named "Purchase Ingredients." Note that each element we add has adjustable attributes. Use the properties panel on the right side of the page to adjust these attributes.
+4. Click on the user task to connect a gateway to it. By clicking the wrench icon on the gateway and declaring it a parallel gateway, we can connect it to two tasks that can happen at the same time: mixing the ingredients, and preheating the oven.
+
+![baking a cake bpmn sample](./img/bake-cake-bpmn.png)
+
+5. Attach the next gateway once these two tasks have completed to move forward.
+6. Add a user task to bake the cake, and finally a user task to ice the cake.
+7. Add an end event, represented by a bold circle, and enjoy your cake!
+
+![completed bpmn diagram](./img/complete-baking-cake-bpmn.png)
+
+:::note
+You can also import a BPMN diagram with Cloud Modeler. Click **Import New Diagram** in the context menu.
+:::
+
+![import diagra,](./img/import-diagram.png)
+
+## Execute your process diagram
+
+:::note
+If a diagram is only saved, it has no effect on your cluster(s). It can be used by all members of your organization.
+
+When you deploy the diagram, it becomes available on the selected cluster and new instances can start.
+:::
+
+To execute your completed process diagram, click **Save**. Then, click **Execute > Save and Deploy**.
+
+You can now start a new process instance to initiate your process. Click the **Execute** dropdown, and then **Start Instance**.
+
+You can now monitor your instances in [Operate](./components/operate/index.md). From your diagram, click the **Execute** dropdown, and **View Process Instances**. This will automatically take you to Camunda Operate to monitor your running instances.
+
+You can also visit an ongoing list of user tasks required in your BPMN diagram. Click the **Execute** dropdown, and **View User Tasks** to automatically be taken to [Tasklist](./components/tasklist/introduction.md).
+
+## Additional resources and next steps
+
+- [Camunda BPMN Tutorial](https://camunda.com/bpmn/)
+- [BPMN Implementation Reference](https://docs.camunda.org/manual/7.16/reference/bpmn20/)
+- [BPMN Engine](https://camunda.com/products/camunda-platform/bpmn-engine/)
```

Comment:
These reminded me we should get a little icon thing for links that pop out of our docs. 💡